easybuild/easyconfigs/s/SuiteSparse/SuiteSparse-5.7.1-intel-2020a-METIS-5.1.0.eb

```
@@ -0,0 +1,19 @@
+name = 'SuiteSparse'
+version = '5.7.1'
+local_metis_ver = '5.1.0'
+versionsuffix = '-METIS-%s' % local_metis_ver
+
+homepage = 'https://faculty.cse.tamu.edu/davis/suitesparse.html'
+description = """SuiteSparse is a collection of libraries manipulate sparse matrices."""
+
+toolchain = {'name': 'intel', 'version': '2020a'}
+toolchainopts = {'unroll': True, 'pic': True}
+
+source_urls = ['https://github.com/DrTimothyAldenDavis/SuiteSparse/archive']
+sources = ['v%(version)s.tar.gz']
```

Please add missing checksums
```suggestion
sources = ['v%(version)s.tar.gz']
checksums = ['5ba5add1663d51a1b6fb128b50fe869b497f3096765ff7f8212f0ede044b9557']
```